docs/next.config.js

```diff
@@ -242,14 +242,22 @@ module.exports = {
   reactStrictMode,
   trailingSlash: true,
   // rewrites has no effect when run `next export` for production
-  async rewrites() {
+  rewrites: async () => {
     return [
       { source: `/:lang(${LANGUAGES.join('|')})?/:rest*`, destination: '/:rest*' },
       // Make sure to include the trailing slash if `trailingSlash` option is set
       { source: '/api/:rest*/', destination: '/api-docs/:rest*/' },
       { source: `/static/x/:rest*`, destination: 'http://0.0.0.0:3001/static/x/:rest*' },
     ];
   },
+  // redirects only take effect in the development, not production (because of `next export`).
+  redirects: async () => [
+    {
+      source: '/x/:path(.{1,})',
+      destination: 'http://0.0.0:3001/x/:path',
+      permanent: false,
+    },
+  ],
```

Comment:
Is it worth to add `toolpad` as well?

Reply:
Yeah, maybe not, I'm not entirely sure it's worth caring about it between MUI Core and MUI X to be honest 😁